docs/src/main/sphinx/connector/ranger.rst

```diff
@@ -0,0 +1,212 @@
+=======================
```

Comment:
It is not a connector but system access control

Reply:
I agree, this should likely live under [/docs/src/main/sphinx/security](https://github.com/trinodb/trino/tree/master/docs/src/main/sphinx/security) in a file called `ranger-access-control.rst`.